spirv_glsl.cpp

```diff
@@ -6664,6 +6664,10 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 		UFOP(transpose);
 		break;
 
+	case OpSRem:
```

Comment:
Are you sure about that? IIRC, SMod has those semantics, but SRem behaves slightly differently I think with negative numbers, so I suspect a workaround function is required, similar to FRem.

Reply:
Sorry, I don't have too much tests for this change.  For my case, SRem can have the same behavior as SMod. 
You are right. We need handle SRem as FRem. Do you think add SRem to FRem is right ? i.e.:
case OpFRem:
case OpSRem:
{
  .......
  break; 
}